.gitignore

```diff
@@ -1,2 +1,3 @@
+.idea
```

Comment:
This should go in your [global gitignore file](https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/)